app/services/flow/flow_state_machine.rb

```diff
@@ -173,6 +182,10 @@ def analytics_optional_step
       'IdV: ' + "#{@analytics_id} optional #{current_step} submitted".downcase
     end
 
+    def attempts_api_submitted_method
+      "idv_#{current_step}_submitted".to_sym
+    end
```

Comment:
I usually find this kind of indirection really difficult to follow, could we add a change directly inside the SSN step?